__tests__/github_service_test.ts

```diff
@@ -115,4 +116,49 @@ describe("Github Tests" , () => {
       })
   });
 
+  it('should return exposed container port', (done: any) => {
```

Comment:
I see a test `should not return exposed container port` for Bitbucket. A similar test for Github is missing.